presto-teradata/repo/com/teradata/terajdbc4/16.20/terajdbc4-16.20.pom

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
any reason not using the jar from the public maven repo?
https://mvnrepository.com/artifact/com.teradata.jdbc/terajdbc4/16.20.00.10

Reply:
@beinan I had tried it out first, this artifact is not present central maven repository.
Even the ICM repo mentioned doesn't have it.

Reply:
Is there any generic JDBC connector that can be leveraged? I couldn't find one